drivers/sn32/rgb_matrix_sn32f248b.c

```diff
@@ -355,15 +324,15 @@ void SN32F24XX_init(void) {
 static void flush(void) {}
 
 void SN32F24XX_set_color(int index, uint8_t r, uint8_t g, uint8_t b) {
-    int corrected_index = led_pos[index];
-    led_state[corrected_index].r = r;
-    led_state[corrected_index].g = g;
-    led_state[corrected_index].b = b;
+    led_state[index].r = r;
+    led_state[index].g = g;
+    led_state[index].b = b;
 }
 
 void SN32F24XX_set_color_all(uint8_t r, uint8_t g, uint8_t b) {
-    for (int i=0; i<DRIVER_LED_TOTAL; i++)
+    for (int i = 0; i < sizeof(led_state) / sizeof(led_state[0]); i++) {
```

Comment:
`sizeof(led_state) / sizeof(led_state[0])` can be changed back to `DRIVER_LED_TOTAL` when `led_state` is made `DRIVER_LED_TOTAL` in size.